apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -12,11 +13,42 @@ export const COLUMNS = {
   NOT_ANSWERED: 3,
 };
 
+const answerOptionsOneFormatter = (percentAnsweredOptionOne, {rowData}) => {
+  return (
+      <MultipleChoiceAnswerCell
+        id={rowData.id}
+        percentValue={rowData.percentAnsweredOptionOne}
+        isCorrectAnswer={rowData.optionOneIsCorrect}
+      />
+  );
+};
+
+const answerOptionsTwoFormatter = (percentAnsweredOptionOne, {rowData}) => {
+  return (
+      <MultipleChoiceAnswerCell
+        id={rowData.id}
+        percentValue={rowData.percentAnsweredOptionTwo}
+        isCorrectAnswer={rowData.optionTwoIsCorrect}
+      />
+  );
+};
+
+const notAnsweredFormatter = (percentAnsweredOptionOne, {rowData}) => {
+  return (
+      <MultipleChoiceAnswerCell
+        id={rowData.id}
+        percentValue={rowData.notAnswered}
+      />
+  );
+};
+
 const questionAnswerDataPropType = PropTypes.shape({
   id: PropTypes.number.isRequired,
   question: PropTypes.string,
   percentAnsweredOptionOne: PropTypes.string,
   percentAnsweredOptionTwo: PropTypes.string,
+  optionOneIsCorrect: true,
```

Comment:
These should be given a `PropType.bool` - since you know it'll be a boolean but you don't know what the value will be.

Reply:
@caleybrock Fixed!  Thanks